adapters/Tapjoy/BuildScript/README.md

```diff
@@ -0,0 +1,25 @@
+# Steps to generate universal static library and framework for iOS Devices and
+Simulators with supported architectures.
```

Comment:
Yes, I have removed my commits. Can you please make changes on your end.
Thanks.